src/Umbraco.Web.UI.Client/src/views/propertyeditors/blocklist/blocklist.component.createButton.controller.js

```diff
@@ -0,0 +1,18 @@
+(function () {
+    "use strict";
+
+    angular
+    .module("umbraco")
+    .controller("Umbraco.PropertyEditors.BlockListPropertyEditor.CreateButtonController", 
+    function BlockListController($scope) {
```

Comment:
Yes, wrong name.. Since its not stored or referenced anywhere it does not need a name. But for making it easier to read i have renamed it to controller